src/js/components/Calendar/doc.js

```diff
@@ -95,12 +101,13 @@ to disable the previous and next buttons.
       For single select, make this the subsequent \`date\` property value.
       For multiple select or ranges, toggle values in \`dates\`.
       Not specifying this property makes the component read only.`),
-    range: PropTypes.bool
+    range: PropTypes.oneOfType([PropTypes.bool, PropTypes.oneOf(['bounds'])])
```

Comment:
I think `bounds` might be a bit confusing here, as Calendar already has a `bounds` property. Some other naming options:
- "uncompressed" - to indicate that `true` implies "compressed" into a single date.
- "always separate", "always distinct", or "always both" - too long but maybe conveys the concept a bit better
- "always" - shortening the previous
- "verbose" - a little odd, but hints at providing further detail
- "long" - similar to previous

Reply:
That makes sense with potential confusion with the `bounds` prop. uncompressed makes most sense to me out of these options, but it's not sitting perfectly well right off the bat. Let me think a little more on some other suggestions.

Reply:
Some thoughts:
- `array` -- just being explicit about the return type?
- `expanded` -- similar to uncompressed mindset
- `unfiltered`
?

Reply:
I'm leaning towards `array`

Reply:
That was my inclination as well.

Reply:
I've updated the value to be called "array".